src/services/swap_service.ts

```diff
@@ -97,23 +96,24 @@ export class SwapService {
             // tslint:disable-next-line:boolean-naming
             skipValidation,
         } = params;
+        let _rfqt;
+        if (apiKey !== undefined && (isETHSell || from !== undefined)) {
+            _rfqt = {
+                ...rfqt,
+                apiKey,
+                // If this is a forwarder transaction, then we want to request quotes with the taker as the
+                // forwarder contract. If it's not, then we want to request quotes with the taker set to the
+                // API's takerAddress query parameter, which in this context is known as `from`.
+                takerAddress: isETHSell ? getContractAddressesForChainOrThrow(CHAIN_ID).forwarder : from || '',
```

Comment:
In the `swap_service` constructor we already fetch the addresses for the chain. We may as well store them and pull `forwarder` off of this.

is `''` an a silent RFQT error vs `undefined` vs `NULL_ADDRESS`?

Reply:
I addressed the address pre-loading in https://github.com/0xProject/0x-api/pull/172/commits/ee23cab5ff5c77aeae3406716f2823d58b218ec9.

There's nothing special about `''`.  Is there a reason to choose `undefined` or `NULL_ADDRESS`?